### PR TITLE
I didn't think a player could be this dumb, but I guess they can be

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: D&B/Repukan is a retard
fix: Apparently the concept that zombies might be a form of bad guy or antag or general valid person is too hard to grasp. This has been clarified in big red letters. Then again, can the people who can't grasp the basic concept of "zombies eat brains and are bad" read this basic english? Maybe I need to dumb it down a little more.
experimental: "Even if you did become a zombie, you still had no directive to kill anyone" - Repukan
/:cl:

### CODE REVIEW

**Comment by kevinz000** [comment]
I don't think you read the ban appeal thread correctly.
In any case, Gee, didn't know my ban would incite a goofball PR!

**Comment by Iamgoofball** [comment]
People play this game to grief and all the fun rounds people talk about are technically bannable

**Comment by AnturK** [comment]
If you really want this fixed and not just spread salt, make this give some "eat brains" or similar objective to the zombie in the zombify proc instead.

**Comment by KorPhaeron** [comment]
Do we really need explicit antag status and clogging up the traitor panel for something thats quite obviously hostile already

**Comment by jughu** [comment]
Hi im a zombie i eat brains
I want to rp zombie.
But i cannot.
Im not antag i get bwoinked.
And goof be angry
Im a opressed zombie.

**Comment by kevinz000** [comment]
@jughu Read. The damn. Thread.
We did NOT BWOINK A ZOMBIE FOR ATTACKING PEOPLE
GOOF JUST DIDNT READ IT.

**Comment by jughu** [comment]
@kevinz im talking from goofs sense

**Comment by Iamgoofball** [comment]
When did Repukan become an admin?

I'm just making sure he doesn't make this mistake again

**Comment by Grazyn** [comment]
I've also seen people revive as zombies, gather their stuff and quietly crawl back to their workstation like nothing happened. Romerol is already hard to come by and zombies pretty easy to fight, throw in "le friendly monster" meme and there's no point buying it ever

**Comment by RemieRichards** [comment]
Solution: when they're transformed start a timer, as it progresses the zombie's limbs fall off (it's rotting), the timer resets whenever they eat a corpse/transform a human/eat a brain.

Simple, if you want to be friendly, you're going to have to convince the station you're worth endless monkey/assistant brains. (also encourages you to actually be hostile)

**Comment by FantasticFwoosh** [comment]
@RemieRichards it'd explicitly have to apply to romerol & zombie organ zombies given that zombie IS a normal variant on halloween

**Comment by RemieRichards** [comment]
@FantasticFwoosh that's fine?

**Comment by Poojawa** [comment]
Request Urban dead lexicon so zombies can sit outside of chemistry yelling "Mrh?"

**Comment by ExcessiveUseOfCobblestone** [comment]
This is a dumb PR, not even because of goof.

**Comment by Armhulen** [comment]
literally just make them all not talk. ZOMBIES FIXED.

**Comment by dannno** [comment]
everything related to this situation is so painfully stupid

**Comment by Qbopper** [comment]
"People play this game to grief" is just objectively false, dude

You admit yourself that you don't play the game, so how can you possibly make judgements like this? Maybe if you just read ban appeals you'd get that impression, but there are a lot of players who enjoy doing things that don't involve griff

**Comment by Iamgoofball** [comment]
I meme the game

**Comment by anconfuzedrock** [comment]
this is actually fucking hilarious goof you do dumb shit a lot but don't let anyone tell you otherwise this is great

**Comment by PKPenguin321** [comment]
changelog is a big stupid meme stop with shitty changelogs

end changelog abuse

File: code/modules/zombie/organs.dm [hunk 1]
```diff
@@ -53,6 +53,7 @@
 		owner << "<span class='narsiesmall'>You can feel your heart stopping, but something isn't right... \
 		life has not abandoned your broken form. You can only feel a deep and immutable hunger that \
 		not even death can stop, you will rise again!</span>"
+		owner << "<span class = 'userdanger'>You are an antagonist.</span>" // because apparently the concept that zombies might be bad guys is too hard to grasp
```
Comment (Dorsisdwarf) [inline code/modules/zombie/organs.dm]: I'm not sure this is clear enough for our players. Perhaps the centcomm announcement font.

Comment (Iamgoofball) [inline code/modules/zombie/organs.dm]: this is bigger

**Comment by optimumtact** [comment]
_**a**_

**Comment by Iamgoofball** [comment]
this needs done though

**Comment by optimumtact** [comment]
smallerprofilerobot

**Comment by Iamgoofball** [comment]
fug
